mat64/matrix_test.go

```diff
@@ -253,6 +256,22 @@ func TestNorm(t *testing.T) {
 	testOneInputFunc(t, "Norm_inf", f, denseComparison, sameAnswerFloatApprox, isAnyType, isAnySize)
 }
 
+func TestNormZero(t *testing.T) {
+	for _, a := range []Matrix{
+		&Dense{},
+		&SymDense{mat: blas64.Symmetric{Uplo: blas.Upper}},
+		&TriDense{mat: blas64.Triangular{Uplo: blas.Upper, Diag: blas.NonUnit}},
+		&Vector{},
+	} {
+		for _, norm := range []float64{1, 2, math.Inf(1)} {
+			panicked, message := panics(func() { Norm(a, norm) })
```

Comment:
Add a closed-over var to capture the result and compare that against expectation.